packages/jest-mock/src/index.ts

```diff
@@ -536,6 +543,7 @@ export class ModuleMocker {
       calls: [],
       instances: [],
       invocationCallOrder: [],
+      lastCall: undefined,
```

Comment:
it should just not be here I think, not set to `undefined` (which might be a return value)